sql/core/src/main/scala/org/apache/spark/sql/execution/joins/HashedRelation.scala

```diff
@@ -780,10 +806,13 @@ private[execution] final class LongToUnsafeRowMap(val mm: TaskMemoryManager, cap
    * Update the address in array for given key.
    */
   private def updateIndex(key: Long, address: Long): Unit = {
+    numKeyLookups += 1
+    numProbes += 1
```

Comment:
hmm, do we need to track the probe time when building the hash relation?

Reply:
@cloud-fan - This is the same behavior for `UnsafeHashedRelation`, while [when it builds hash relation](https://github.com/apache/spark/blob/master/sql%2Fcore%2Fsrc%2Fmain%2Fscala%2Forg%2Fapache%2Fspark%2Fsql%2Fexecution%2Fjoins%2FHashedRelation.scala#L479), it [updates the lookup/probe metrics as well](https://github.com/apache/spark/blob/master/core%2Fsrc%2Fmain%2Fjava%2Forg%2Fapache%2Fspark%2Funsafe%2Fmap%2FBytesToBytesMap.java#L528-L533). I guess it would be good to keep consistent between `UnsafeHashedRelation` and `LongHashedRelation` here?